docs/content/concepts/layer/control/catalog/sp800-53rev5-example.md

```diff
@@ -0,0 +1,340 @@
+>
+> c. Review and update the current access control:
+>   1. Policy *[Assignment: organization-defined frequency]* and following *[Assignment: organization-defined events]*; and
+>   2. Procedures *[Assignment: organization-defined frequency]* and following *[Assignment: organization-defined events]*.
+>
+> __Discussion__: Access control policy and procedures address the controls in the AC family that are implemented within systems and organizations. The risk management strategy is an important factor in establishing such policies and procedures. Policies and procedures contribute to security and privacy assurance. Therefore, it is important that security and privacy programs collaborate on the development of access control policy and procedures. Security and privacy program policies and procedures at the organization level are preferable, in general, and may obviate the need for mission- or system-specific policies and procedures. The policy can be included as part of the general security and privacy policy or be represented by multiple policies reflecting the complex nature of organizations. Procedures can be established for security and privacy programs, for mission or business processes, and for systems, if needed. Procedures describe how the policies or controls are implemented and can be directed at the individual or role that is the object of the procedure. Procedures can be documented in system security and privacy plans or in one or more separate documents. Events that may precipitate an update to access control policy and procedures include assessment or audit findings, security incidents or breaches, or changes in laws, executive orders, directives, regulations, policies, standards, and guidelines. Simply restating controls does not constitute an organizational policy or procedure.
+>
+> __Related Controls__: IA-1, PM-9, PM-24, PS-8, SI-12.
+>
+> __Control Enhancements__: None.
+>
+> __References__: [OMB A-130], [SP 800-12], [SP 800-30], [SP 800-39], [SP 800-100], [IR 7874].
+
+Encoding this text in OSCAL entails an analysis that turns it "inside out" and exposes the implicit structures (the "bones") as a kind of exoskeleton around the text, using a common and non-proprietary data encoding standard such as XML or JSON to provide the "form" of the cast. Within this organization, all these data points become distinguishable and recognizable to any downstream processor (consumer or partner) who expects OSCAL, or who can interpret it, defining OSCAL both narrowly, in its technical senses, and broadly: as a standard, a set of norms and conventions, a collection of tools, applications and resources, and a community of practice. From the encoded data, the text's "original format" (or something like it) can be restored, but much else can be done as well: the semantics of the information (the set of meaningful discriminations) can be mined, sifted, collated, ranked, annotated, and related to other data points internal or external.
```

Comment:
We should be more direct and use less exposition in this paragraph. Something like:
```suggestion
Representing this text in a machine readable form involves providing more structure around the text, allowing each data point to be easily identified by a machine. OSCAL defines the syntax for XML, JSON, and YAML structures to represent this information, allowing any downstream consumer, who expects OSCAL, to parse this data. This allows a processor to produce the text's "original" or any other format from the structured OSCAL representation. The processor can also sort, analyze, filter, enrich or transform this data for any purpose.
```